jobs/firstinqueue-priming.groovy

```diff
@@ -110,7 +110,17 @@ def primeWorkers() {
    def jobs = [
        "sync-webapp": { prime() },
    ];
-   for (def i = 0; i < NUM_WORKER_MACHINES; i++) {
+
```

Comment:
@csilvers I ended up moving the increment logic inside the `primeWorkers` function so we always make sure to include that extra worker (in case someone runs this job manually).

Let me know if you find it ok, or if I should make any further changes.

Reply:
Yeah, I can see argument either way.  Your call!